_config.yml

```diff
@@ -13,7 +13,7 @@ twitter_username: rackspace
 github_username:  rackerlabs
 permalink: /:categories/:slug
 # navigation: ['Style', 'Layout', 'Components', 'Templates']
-exclude: ['Gemfile', 'Gemfile.lock','README.md', 'vendor', 'node_modules', '_site']
```

Comment:
This may still be necessary. Double check if you need to do any advanced exclusion based on feedback in `.gitignore`.

Reply:
Unfortunately, the `@` in the folder name for the toolkit is causing the Jekyll config to hiccup when I try more advanced exclusions, and escaping it doesn't seem to help. Will flag for a follow-up.